mandatory/3-playing-computer.js

```diff
@@ -9,8 +9,8 @@
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
+  4. How many times is "f1" called?    //3 times
+  5. How many times is "f2" called?    //2 times
```

Comment:
It's actually the other way around. It is two times for f1 and 3 times for f2. f2 gets called when i%2 == 0, which is true for 0,2, and 4. And f1 gets called for all of the other cases which is true for 1 and 3.